mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,15 +1,25 @@
 import {useEffect, useState} from "react";
-import {flexRender, getCoreRowModel, useReactTable} from "@tanstack/react-table";
+import InProgress from "../inProgress/InProgress";
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
 import {columns} from "src/component/table/columns";
 import {DayReport} from "src/model/businessModel/DayReport";
 import {DayReportService} from "src/service/DayReportService";
 import styles from "src/component/table/Table.module.scss";
 
 export const Table = () => {
   const [data, setData] = useState<DayReport[]>([]);
+  const [loading, setLoading] = useState(true);
 
   useEffect(() => {
     DayReportService.onValueFromRealTimeDb(setData);
+    setTimeout(() => {
+      setLoading(false);
+    // eslint-disable-next-line no-magic-numbers
```

Comment:
don't use disable of eslint rules. If we will use disable then why we need eslint))